res/values/strings.xml

```diff
@@ -576,11 +575,11 @@
     <string name="emoji_search_results">Search results</string>
     <string name="emoji_not_found">No emoji found</string>
     <string name="emoji_recent">Recently used</string>
-    <string name="emoji_people">People &amp; Body</string>
-    <string name="emoji_nature">Animals &amp; Nature</string>
-    <string name="emoji_foods">Food &amp; Drink</string>
+    <string name="emoji_people">People and Body</string>
+    <string name="emoji_nature">Animals and Nature</string>
+    <string name="emoji_foods">Food and Drink</string>
```

Comment:
i think, the reason for the ampersand was to save some space on the tabs and to have less text overall there.